### PR TITLE
FFMT-2139 - P and D email templates

### PR DESCRIPTION
**Pull request by sneha-nerekar**
Covered 
1. Expected Delivery Date Update
2. Items Ready for Delivery
 email templates

### CODE REVIEW

File: labels/en-US.json [hunk 1]
```diff
@@ -238,6 +238,21 @@
   "deleteQuoteName": "Delete Quote",
   "deliverTo": "Deliver To",
   "deliverYourItem": "Thanks for letting us know you're here! Please fill out the questions below, and a store associate will be right out with your items.",
+  "deliveryHello" : "Hello",
+  "deliveryReadyHeader":"Items Ready for Delivery",
+  "deliveryReadyBody": "Item(s) in your order are ready to be delivery. A member of out team will contact you to schedule the delivery",
```
Comment (PapaTaylor) [inline labels/en-US.json]: Typos. Should be more like this:
"Item(s) in your order are ready to be delivered. A member of our team will contact you to schedule the delivery."

**Review by PapaTaylor — CHANGES_REQUESTED** [review]
Please see my comment

**Comment by sneha-nerekar** [comment]
Changed the line as per wireframe "**Item(s) in your order are ready to be delivery.  A member of our team will contact you to schedule the delivery**" also tried to change accordingly in de.json but the line is good there as per google translation, hence no changes required.

File: labels/en-US.json [hunk 2]
```diff
@@ -238,6 +238,21 @@
   "deleteQuoteName": "Delete Quote",
   "deliverTo": "Deliver To",
   "deliverYourItem": "Thanks for letting us know you're here! Please fill out the questions below, and a store associate will be right out with your items.",
+  "deliveryHello" : "Hello",
+  "deliveryReadyHeader":"Items Ready for Delivery",
+  "deliveryReadyBody": "Item(s) in your order are ready to be delivery. A member of our team will contact you to schedule the delivery",
```
Comment (PapaTaylor) [inline labels/en-US.json]: Hi @sneha-nerekar,
There is still an issue with the message. The word, "delivery" in the first sentence should be, "delivered"

FROM: "Item(s) in your order are ready to be delivery. A member of our team will contact you to schedule the delivery"

TO: "Item(s) in your order are ready to be deliver**ed**. A member of our team will contact you to schedule the delivery"

**Review by PapaTaylor — CHANGES_REQUESTED** [review]
Please see my comment.

File: labels/en-US.json (hunk 2 above)
Comment (sneha-nerekar) [reply labels/en-US.json]: Hi @PapaTaylor, changed the the word, "delivery" to "delivered"

**Review by PapaTaylor — APPROVED** [review]

**Pull request merged**